index.js

```diff
@@ -102,3 +102,27 @@ io.on('connection', socket => {
         console.log(`${currentUser} score: ${score}`);
     });
 });
+
+//Functions to perform fullscreen and chat toggle
+function goFullScreen() {
```

Comment:
Please stick to the ES6+ syntax of the whole repository as well when writing functions and declaring variables.